### PR TITLE
Update to Java 16

### PR DESCRIPTION
**Pull request by Seggan**
## Changes
Updated to Java 16 since we waiting for MC 1.17 anyway. Also added some more launch messages

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break world generation, mob spawning, and the like.
- [x] I followed the existing code standards and didn't mess up the formatting.

### CODE REVIEW

**Comment by Seggan** [comment]
We'll also need o update InfinityLib. I tried, but Riley you removed so much :NotLikeThis:

File: src/main/java/io/github/addoncommunity/galactifun/api/universe/attributes/atmosphere/AtmosphericEffect.java [hunk 1]
```diff
@@ -7,26 +7,23 @@
...
+ *
  * @author Mooy1
  */
-public final class AtmosphericEffect {
-    
-    @Nonnull
-    private final EffectType type;
-    private final int level;
-    
+public record AtmosphericEffect(
```
Comment (Mooy1) [inline src/main/java/io/github/addoncommunity/galactifun/api/universe/attributes/atmosphere/AtmosphericEffect.java:13]: Why is this a record?

**Comment by Mooy1** [comment]
Ill fix up the InfinityLib stuff, just cleaning up stuff that i noticed wasn't being used much.

**Comment by Mooy1** [comment]
Also i think the build is failing because the build with maven github action is targeting 1.8

File: src/main/java/io/github/addoncommunity/galactifun/api/universe/attributes/atmosphere/AtmosphericEffect.java (hunk 1 above)
Comment (Seggan) [reply src/main/java/io/github/addoncommunity/galactifun/api/universe/attributes/atmosphere/AtmosphericEffect.java:13]: intellij... ill rework it anyway so it wont stay long

**Pull request merged**